install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -66,6 +66,11 @@ the master node. Do this by setting the
 |link:https://access.redhat.com/documentation/en/red-hat-gluster-storage/3.1/single/container-native-storage-for-openshift-container-platform/[Container Native Storage with GlusterFS]
 |Container Native Storage (CNS) utilizes heketi to manage Gluster Storage
 
+|Ceph RBD
+|`kubernetes.io/rbd`
+|xref:../../install_config/configuring_openstack.adoc#install-config-configuring-openstack[Configuring for OpenStack]
```

Comment:
@adellape , well, it's an obvious mistake, however, I have no idea where the Ceph required configuration is described.

@rootfs , please, do you know where the Ceph required configuration is described?